library/src/main/java/com/novoda/downloadmanager/LiteBatchFileBuilder.java

```diff
@@ -40,20 +40,60 @@ public BatchFileBuilder saveTo(String path) {
 
     @Override
     public BatchFileBuilder saveTo(String path, String fileName) {
-        this.path = Optional.fromNullable(path).filterNot("/"::equals);
```

Comment:
We thought we fixed it by filtering out paths that were `/` only, but this wouldn't solve the issue with a `path` that **ends** in `/` (e.g. `my/path/`).